packages/flutter/lib/src/services/text_input.dart

```diff
@@ -3062,5 +3062,32 @@ final class IOSSystemContextMenuItemDataShare extends IOSSystemContextMenuItemDa
   }
 }
 
+/// A [IOSSystemContextMenuItemData] for the system's built-in Live Text
+/// (OCR) button.
+///
+/// This button is only available on iOS 15.0+ devices with camera support.
+///
+/// The title and action are both handled by the platform. When [title] is not
+/// provided, the platform will use its default localized title.
+///
+/// See https://github.com/flutter/flutter/issues/169781
+final class IOSSystemContextMenuItemDataLiveText extends IOSSystemContextMenuItemData
+    with Diagnosticable {
+  /// Creates an instance of [IOSSystemContextMenuItemDataLiveText].
+  const IOSSystemContextMenuItemDataLiveText({this.title});
+
+  @override
+  final String? title;
```

Comment:
as discussed in my previous message, title isn't needed here. Please follow the example of copy/paste/select actions.